vue-components/src/components/TrameListBrowser.js

```diff
@@ -55,14 +55,26 @@ const QUERY =
 
 export default {
   props: {
+    showPathWithIcon: {
+      type: Boolean,
+      default: false
+    },
     pathIcon: {
       type: String,
       default: "mdi-folder-outline",
     },
+    showIcon: {
+      type: Boolean,
+      default: true
```

Comment:
Usually default boolean props are set to false. Maybe you want to replace that prop to `hideIcon` instead.

The reason behind is when you write the template, you would do it like `<list-browser hide-icon />` otherwise doing the following will be a noop `<list-browser show-icon />.